var/spack/repos/builtin/packages/arm-perflibstools/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class ArmPerflibstools(MakefilePackage):
+    """This project provides tools to enable users of HPC applications to understand
+    which routines from Arm Performance Libraries are being called."""
+
+    homepage = "https://github.com/ARM-software/perf-libs-tools"
+    git      = "https://github.com/ARM-software/perf-libs-tools.git"
+
+    # notify when the package is updated.
+    maintainers = ['OliverPerks', 'chrisgoodyer']
+
+    version('master', branch='master')
+
+    depends_on('python@3:')
+
+    # Patch to remove xblas.h include and non gcc routines
+    patch('simple.patch')
```

Comment:
Is this patch already available for download somewhere? If so, it's preferable to use a `url` in the patch directive and avoid storing the patch in the builtin repo. If it's not available anywhere, then the recipe is fine like it is.

Reply:
It's not at the moment, I'm just checking to see if I can make a branch on the project repo, which would save us having to do this.

Reply:
@OliverPerks Ok, let me know.

Reply:
Is there any chance that https://github.com/ARM-software/perf-libs-tools/pull/4 gets merged?